Fixes - Issue #28

siddharthashw opened the pull request:
Fixes - #28 

**1. IStringConstant.cs** - removed Test cases string constants
**2. StringConstant.cs** - removed Test cases string constants and used environment variable for clientId and clientSecret
**3. IStringConstantTest.cs** - added test cases string constant
**4. StringConstantTest.cs** - added test cases string constant
**5. BaseProvider.cs** - register IStringConstantTest
**6. IntegrationBaseProvider.cs** - register IStringConstantTest
**7. ProjectModuleIntegrationTest.cs** - used StringConstantTest instead of StringConstant
**8. ProjectModuleTest.cs** - used StringConstantTest instead of StringConstant
**9. UserModuleIntegrationTest.cs** - used StringConstantTest instead of StringConstant
**10. UserModuleTest.cs** - used StringConstantTest instead of StringConstant

siddharthashw committed:
StringConstant.cs - removed clientId and clientSecret credential from the code and set them to environment variable
IntegrationBaseProvider.cs - used environment varibale for clientId and clientsecret

Fixes - #28

siddharthashw committed:
StringConstant.cs - removed clientId and clientSecret credential from the code and set them to environment variable
IntegrationBaseProvider.cs - used environment varibale for clientId and clientsecret

Fixes - #28

siddharthashw committed:
Merge branch 'Issue-#28' of https://github.com/Promact/oauth-client.git

chintans commented on Promact.OAuth.Client/src/Promact.OAuth.Client.Test/IntegrationBaseProvider.cs:
```diff
@@ -33,8 +33,8 @@ public IntegrationBaseProvider()
             _discoveryClient = discovery.GetAsync().Result;
-            _client = new TokenClient(_discoveryClient.TokenEndpoint, _stringConstant.PromactOAuthClientId, 
-                _stringConstant.PromactOAuthClientSecret);
+            _client = new TokenClient(_discoveryClient.TokenEndpoint, Environment.GetEnvironmentVariable(_stringConstant.PromactOAuthClientId), 
```
This file required no change

siddharthashw replied:
ok

chintans commented on Promact.OAuth.Client/src/Promact.OAuth.Client/Util/StringConstant/StringConstant.cs:
```diff
@@ -428,7 +428,7 @@ public string PromactOAuthClientId
             get
             {
-                return "OOKK4SVDYQ8XRUU";
+                return "PromactOAuthClientId";
```
Test related constants should be in Test project. here you could return Environment.GetVariable

siddharthashw replied:
ok

siddharthashw committed:
IStringConstant.cs - removed Test cases string constants
StringConstant.cs - removed Test cases string constants
IStringConstantTest.cs - added test cases string constant
StringConstantTest.cs - added test cases string constant
BaseProvider.cs - register IStringConstantTest
IntegrationBaseProvider.cs - register IStringConstantTest
ProjectModuleIntegrationTest.cs - used StringConstantTest instead of StringConstant
ProjectModuleTest.cs - used StringConstantTest instead of StringConstant
UserModuleIntegrationTest.cs - used StringConstantTest instead of StringConstant
UserModuleTest.cs - used StringConstantTest instead of StringConstant

Fixes - #28
